docs/User/SWCI_APIs.md

```diff
@@ -29,16 +29,17 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`getTaskRunnerInfo()`|This method prints the current status of the specified Taskrunner ID.|`trName`|
 |`getTaskRunnerPeerStatus()`|This method prints the current status for the specified PEER (SWOP node) in the Taskrunner’s context.|`trName`, `idx`|
 |`getTaskRunnerStatus()`|This method prints the overall status of the taskrunner.|`trName`|
+|`getTaskrunnerStatus() <trName:string>`|Displays the current status of the specified Taskrunner.<br> `trName`: Specifies the unique name of Taskrunner.<br> Provides below information like,<br>- TASK NAME – Current running task or ‘Empty’ if no task is assigned.<br>- PEER TYPE – SL for RUN_SWARM tasks, SWOP for all other tasks.<br>- TASK STATE – Current task state.<br>- ACTIVE, COMPLETED and FAILED PEERS information.<br>- TIME STAMPs of various events on the TASKRUNNER contract.| |
```

Comment:
I see 2 getTaskrunnerStatus()..  Remove the 1st one..

Also all the arguments and its description, can be moved to the 3rd column.